content/events/2020/trivia/contents.lr

```diff
@@ -0,0 +1,16 @@
+title: Trivia Day
+---
+body:
+
+# TRIVIA DAY
+We are having a trivia day. The trivia will be done over a 24-hour period to ensure that people in all time zones can participate. Points will be awarded on correct answers and can be done unlimited times. 
```

Comment:
```suggestion
We are having a trivia day. The trivia will be done over a 24-hour period to ensure that people in all time zones can participate. Points will be awarded on correct answers.
```
There's an upwards limit on the # of questions that can be done in a trivia session, might as well just remove the last bit